tests/br_debug_meta/run.sh

```diff
@@ -15,24 +15,25 @@
 
 set -eu
 DB="$TEST_NAME"
-TABLE="usertable1"
+TABLE="usertable"
 
 run_sql "CREATE DATABASE $DB;"
+go-ycsb load mysql -P tests/$TEST_NAME/workload -p mysql.host=$TIDB_IP -p mysql.port=$TIDB_PORT -p mysql.user=root -p mysql.db=$DB
+run_sql "split table $DB.$TABLE by ('user6282602628620641459'), ('user6282603728132269670'), ('user6282604827643897881'), ('user6282610325202038936');"
 
-run_sql "CREATE TABLE $DB.$TABLE( \
-  YCSB_KEY varchar(64) NOT NULL, \
-  FIELD0 varchar(1) DEFAULT NULL, \
-  PRIMARY KEY (YCSB_KEY) \
-) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin;"
-
-run_sql "INSERT INTO $DB.$TABLE VALUES (\"a\", \"b\");"
-run_sql "INSERT INTO $DB.$TABLE VALUES (\"aa\", \"b\");"
+table_region_sql="SELECT COUNT(*) FROM information_schema.tikv_region_status WHERE db_name = '$DB' AND table_name = '$TABLE';"
+for i in $(seq 10); do
+    regioncount=$(run_sql "$table_region_sql" | awk '/COUNT/{print $2}')
+    [ $regioncount -ge 5 ] && break
+    sleep 3
+done
+run_sql "$table_region_sql"
```

Comment:
`SPLIT TABLE` should be effective immediately, so no need to check region count again 🤔 (you could check the `TOTAL_SPLIT_REGION` field returned by `SPLIT TABLE` if you need to be absolutely sure)

Reply:
It does split, however, data is not distributed to multiple regions, so I think just leave the loop and let TiKV splits.

Reply:
ok.